apps/web/app/[locale]/kanban/page.tsx

```diff
@@ -36,116 +33,144 @@ const Kanban = () => {
 
 	const teamMembers: IImageOverlapper[] = [];
 
-	activeTeamMembers.map((member: any)=> {
+	activeTeamMembers.map((member: any) => {
 		teamMembers.push({
 			id: member.employee.user.id,
 			url: member.employee.user.imageUrl,
 			alt: member.employee.user.firstName
-		})
+		});
 	});
+	console.log('data-kanban', data);
+	const tabs = [
+		{ name: 'Today', value: KanbanTabs.TODAY },
+		{ name: 'Yesterday', value: KanbanTabs.YESTERDAY },
+		{ name: 'Tomorrow', value: KanbanTabs.TOMORROW }
+	];
+	// eslint-disable-next-line react/no-unstable-nested-components
+	const Label = ({ active, selected }: { active: string; selected: string }) => (
+		<div
+			style={{
+				fontWeight: selected ? 'bold' : 'normal'
+			}}
+			className="text-left"
+		>
+			{active}
+		</div>
+	);
 
+	const sampleDropdownItem = (key: string) => {
+		const data = {
+			key: '1',
+			Label: Label,
+			selectedLabel: <strong>{key}</strong>,
+			itemTitle: key
+			// other properties can be added as needed
+		};
+		return data;
+	};
 	return (
 		<>
-			<MainLayout
-				showTimer={true}
-			>
-				<div className={'fixed flex flex-col bg-white dark:bg-dark--theme h-auto z-10 px-[32px] mx-[0px] w-full'}>
+			<MainLayout showTimer={true}>
+				<div className={' flex flex-col bg-white dark:bg-dark--theme h-auto z-10 px-[32px] mx-[0px] w-full'}>
 					<div className="flex flex-row items-center justify-between mt-[34px]">
 						<Breadcrumb paths={breadcrumbPath} className="text-sm" />
-						<div className="flex flex-row items-center gap-[12px]">
-							{activeTeamMembers.length > 0 ? 
-								<p>08:00 ( UTC +04:30 )</p>
-							:
-								<Skeleton height={20} width={120} borderRadius={5} className="rounded-full dark:bg-[#353741]" />
-							}
-							<VerticalLine /> 
-								<ImageOverlapper images={teamMembers}/>
-							<VerticalLine />
+					</div>
+					<div className="flex justify-between items-center mt-10">
+						<h1 className="text-4xl font-semibold ">Kanban Board</h1>
```

Comment:
@Anishali2 we need to use translation files for each text like "Kanban Board" (see other components)

Reply:
Ohh I forget to add will fix that tomorrow

Reply:
Sir if there is any text that are not exist in translation then i have to add in every langauge json ?

Reply:
@Anishali2 think so (check in all places etc)

Reply:
Yes there are some text also that are not used by translation i can add these also into translation and consider this change in future also